src/items/Camera.java

```diff
@@ -0,0 +1,29 @@
+package items;
+
+import worldofzuul.Game;
+import pointsofinterest.PointOfInterest;;
+
+public class Camera extends Item {
+    public Camera (Game game) {
+        super("camera", "A digital camera", game);
+    }
+
+    @Override
+    public void use () {
+        PointOfInterest pointOfInterest = game.getCurrentPointOfInterest();
+
+        // If the poi is already fixed, then don't allow the item to be used (it would keep adding " (with photo)" to the name)
+        if (!game.getCurrentRoom().getName().equals("factory") || !pointOfInterest.getName().equals("map") || pointOfInterest.isFixed()) {
```

Comment:
Vi burde nok kalde dette point of interest for noget andet, map giver ikke rigtigt nogen mening

Reply:
Enig, vi kunne evt kalde det poster? :)

Reply:
Altså for at være realistisk så skal det måske være et hemmeligt dokument eller noget i den stil